gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/util/ServerGremlinExecutor.java

```diff
@@ -100,9 +101,28 @@ public ServerGremlinExecutor(final Settings settings, final ExecutorService grem
      */
     public ServerGremlinExecutor(final Settings settings, final ExecutorService gremlinExecutorService,
                                  final T scheduledExecutorService, final Class<T> scheduleExecutorServiceClass,
-                                 final GraphManager graphManager) {
+                                 GraphManager graphManager) {
         this.settings = settings;
 
+        if (null == graphManager) {
```

Comment:
looks like there's some indentation problems in the following lines of code. should be four spaces between the curly brackets.